docs/dev/tutorials/cross-chain-tutorial.md

```diff
@@ -1,64 +1,115 @@
 # Cross-chain governance
 
-This tutorial serves as an example of how to implement L1 to L2 contract interaction. The following functionality is implemented in this tutorial:
+This tutorial shows you how to implement communication between L1 and L2 with the following example:
 
-- A "counter" smart contract is deployed on zkSync, which stores a number that can be incremented by calling the `increment` method.
-- A "governance" smart contract is deployed on layer 1, which has the privilege to increment the counter on zkSync.
+- A **governance** smart contract is deployed on layer 1. This contract has a function that can request a transaction on zkSync layer 2.
+- A **counter** smart contract is deployed on zkSync layer 2. The contract stores a number that can be incremented by calling the `increment` method. The goverance contract on layer 1 calls this function.
 
+## Prerequisites
 
+- You are already familiar with deploying smart contracts on zkSync. If not, please refer to the first section of the [quickstart tutorial](../building-on-zksync/hello-world.md).
+- You already have some experience working with Ethereum.
+- You have a web3 wallet app which holds some Goerli test ETH and some zkSync test ETH.
 
+:::warning
+- The `yarn` instructions only run on Node version 14.
+- Use [`nvm`](https://github.com/nvm-sh/nvm) to switch to Node 14 with the command `nvm use 14`.
+:::
+
+## Project structure
 
-::: warning
+Open a terminal window, create a new folder for the project tutorial, e.g. `mkdir cross-chain-tutorial`, and `cd` into the folder.
 
-Please note that breaking changes were introduced in `zksync-web3 ^0.13.0`. The API layer now operates with `gas` and the `ergs` concept is only used internally by the VM. 
+Now create separate folders to store contracts and scripts on L1 and L2.
 
-This tutorial will be updated shortly to reflect those changes.
+```sh
+mkdir L1-governance 
+mkdir L2-counter
+```
 
+::: note
+- The `L1-governance` code is a default Hardhat project used to deploy a contract on L1.
+- The `L2-counter` code includes all zkSync dependencies and configurations for L2.
 :::
 
-## Preliminaries
+## L1 governance
 
-In this tutorial, it is assumed that you are already familiar with deploying smart contracts on zkSync. If not, please refer to the first section of the [quickstart tutorial](../building-on-zksync/hello-world.md).
+1. `cd` into `L1-governance`.
 
-It is also assumed that you already have some experience working with Ethereum.
+2. Initialize the project:
 
-## Project structure
+::: code-tabs
```

Comment:
These steps are not necessary.

Reply:
from the comment, it's not clear which steps you're referring to ... can you use the suggestion tool please by highlighting the text you're talking about